README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Bohan Yin's Biography
+
+Hi, my name is **Bohan Yin**, a second-year MPP student from Uchicago.  I am interested in data science and international relations (*although they seem unrelated*). During my spare time, I do a lot of things:
+> *Playing Tennis
+> Cooking and Baking
+> Playing video games
+> etc...*
+
```

Comment:
This looks nice. But how about trying to make a list? That would also work and it would look better in the final version IMHO.